doc/modules/array_api.rst

```diff
@@ -9,7 +9,12 @@ Array API support (experimental)
 The `Array API <https://data-apis.org/array-api/latest/>`_ specification defines
 a standard API for all array manipulation libraries with a NumPy-like API.
 Scikit-learn's Array API support requires
-`array-api-compat <https://github.com/data-apis/array-api-compat>`__ to be installed.
+`array-api-compat <https://github.com/data-apis/array-api-compat>`__ to be installed,
+and the environment variable `SCIPY_ARRAY_API` must be set to `1`:
+
+.. prompt:: bash $
+
+   SCIPY_ARRAY_API=1
```

Comment:
Use `export` as in the [scipy doc](https://docs.scipy.org/doc/scipy/dev/api-dev/array_api.html#using-array-api-standard-support):
```suggestion
   export SCIPY_ARRAY_API=1
```
One way it can matter because `export` also affect subprocess
```
SCIPY_ARRAY_API=1
# without export SCIPY_ARRAY_API is not seen as set in python script (i.e. subprocess of the current shell process)
python script_using_array_api.py
```